elements/pf-progress/pf-progress.css

```diff
@@ -0,0 +1,152 @@
+.container {
+  --pf-c-progress--GridGap: var(--pf-global--spacer--md, 1rem);
```

Comment:
this and others are listed in the jsdoc, but if we set them here on the container, they won't be user-configurable, so they should probably be fallbacks instead, unless patternfly's implementation works like this